viper-toolchain-versions.sh

```diff
@@ -8,3 +8,4 @@
 SILVER_REF="f645539a04e76d70337d7b5b108e7f9a9de9bc7a"
 SILICON_REF="v.21.07-release"
 CARBON_REF="v.21.07-release"
+VIPERSERVER_REF="master"
```

Comment:
Is there any reason not to use version "v.21.07-release" like with the other dependencies?

Reply:
There will be after https://github.com/viperproject/viperserver/pull/44 is merged. This then needs to be changed to the new commit